charts/flyte-core/templates/common/ingress.yaml

```diff
@@ -1,95 +1,24 @@
 {{- define "grpcRoutes" -}}
-{{- $grpcPort := 81 -}}
-{{- if eq .Values.configmap.adminServer.server.security.secure true -}}
-  {{- $grpcPort = 80 -}}
-{{- end }}
 # NOTE: Port 81 in flyteadmin is the GRPC server port for FlyteAdmin.
-- path: /flyteidl.service.SignalService
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.SignalService/*
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.AdminService
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.AdminService/*
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.DataProxyService
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.DataProxyService/*
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.AuthMetadataService
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.AuthMetadataService/*
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.IdentityService
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /flyteidl.service.IdentityService/*
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /grpc.health.v1.Health
-  pathType: ImplementationSpecific
-  backend:
-    service:
-      name: flyteadmin
-      port:
-        number: {{ $grpcPort }}
-- path: /grpc.health.v1.Health/*
+{{- $paths := (include "flyteadmin.ingress.grpcPaths" .) | fromYamlArray }}
+{{- range $path := $paths }}
+- path: {{ $path }}
+  {{- if semverCompare ">=1.18-0" $.Capabilities.KubeVersion.GitVersion }}
```

Comment:
I'm not quite sure `Capabilities.KubeVersion.GitVersion` is still supported on Helm 3 ([ref](https://helm.sh/docs/chart_template_guide/builtin_objects/))